train.py

```diff
@@ -30,6 +30,9 @@
 SAMPLE_SIZE = 100000
 L2_REGULARIZATION_STRENGTH = 0
 SILENCE_THRESHOLD = 0.3
+ADAM_OPTIMIZER = 'adam'
+SGD_OPTIMIZER = 'sgd'
+DEFAULT_SGD_MOMENTUM = 0.9
```

Comment:
We can call it `SGD_MOMENTUM`, as none of the other defaults have a `DEFAULT` prefix